src/nord.yml

```
@@ -26,7 +26,7 @@ colors:
     matches:
       foreground: CellBackground
       background: '#88c0d0'
-    bar:
+    footer_bar:
```

To ensure to not break the setups of these users only add documentation to the `colors.search.bar` key about the deprecation and leave the key as is.

```suggestion
    # This key is deprecated as of Alacritty version 0.11.0 and will be removed accordingly within the next versions.
    # See https://github.com/alacritty/alacritty/releases/tag/v0.11.0 for details.
    bar:
```